### DISCUSSION TITLE
Question on Lattigo's cross platform support

### DISCUSSION THREAD

**Discussion by Guy-Moshkowich** [tuneinsight/lattigo#525] [Q&A]
Does Lattigo support cross platform serialization? more specific, if we serialize a ciphertext in a big-endian machine - can we de-serialize it in a little-endian machine and vice versa?

**Comment by Pro7ech** [comment]
All serialization in the library is done using the `utils/buffer` package, which explicitely uses `LittleEndian` for both read and write operations (see [/buffer/writer.go](https://github.com/tuneinsight/lattigo/blob/main/utils/buffer/writer.go) and [/buffer/reader.go](https://github.com/tuneinsight/lattigo/blob/main/utils/buffer/reader.go)).
